src/test/resources/Unformatted Strings.txt

```diff
@@ -0,0 +1,236 @@
+
```

Comment:
Please add a test case with input containing indentation. This should be ignored by the `NULL` formatter and the formatting should be replaced (not added) by the other formatters.

Reply:
I agree the testcases need to be improved. Would random indentation (between 1 and lets say 32) spaces be enough to determine if not only adding but correcting indentation works correctly ?

Reply:
It would be sufficient if there was some code with different indentations (tabs, and various amounts of spaces, correct and wrong indentation).

Reply:
I will not generate the indentation on the fly by the tester but change the file itself.